lib/px/utils/pxtimer.lua

```diff
@@ -17,25 +17,42 @@ function M.application(file_name)
 	local ngx_timer_at = ngx.timer.at
 
 	function send_initial_enforcer_telemetry()
-		if config == nil or config.px_appId == 'PX_APP_ID' then
+		if config == nil or not config.px_enabled then
+			px_logger.debug("module is disabled, shutting down enforcer telemetry")
```

Comment:
I think `skipping` is better phrasing than `shutting down`.

Reply:
👍